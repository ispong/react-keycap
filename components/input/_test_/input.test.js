import React from 'react'
import Input from '../input'
import ReactDOM from 'react-dom'

test('test input component', () => {
  const mountNode = document.createElement('div')

  const inputNode = (
    <Input cap={'default'}/>
  )

  ReactDOM.render(inputNode, mountNode)
})
