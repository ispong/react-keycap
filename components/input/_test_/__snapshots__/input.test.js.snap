// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button component 1`] = `
<input
  className="keycap-input keycap-input-default"
  cap="default"
>
  test
</input>
`;
